eng/scripts/automation_init.sh

```diff
@@ -57,4 +56,7 @@ cat > $outputFile << EOF
     "GOROOT": "$GOROOT"
   }
 }
-EOF
\ No newline at end of file
+EOF
+
```

Comment:
can you update line#21 to `wget -q https://golang.org/dl/go1.22.0.linux-amd64.tar.gz` to suppress its output?